[2024-05-18T13:06:04.443+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-18T13:06:04.479+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-18T13:06:04.481+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-05-18T13:06:04.523+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): seceond_task> on 2024-05-01 00:00:00+00:00
[2024-05-18T13:06:04.547+0000] {standard_task_runner.py:60} INFO - Started process 132 to run task
[2024-05-18T13:06:04.567+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'seceond_task', 'scheduled__2024-05-01T00:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/frist-dag.py', '--cfg-path', '/tmp/tmpxao60ruu']
[2024-05-18T13:06:04.570+0000] {standard_task_runner.py:88} INFO - Job 285: Subtask seceond_task
[2024-05-18T13:06:04.669+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.seceond_task scheduled__2024-05-01T00:00:00+00:00 [running]> on host 7b23a6949395
[2024-05-18T13:06:04.931+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='seceond_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-01T00:00:00+00:00'
[2024-05-18T13:06:04.935+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-18T13:06:04.944+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo task 2']
[2024-05-18T13:06:04.981+0000] {subprocess.py:86} INFO - Output:
[2024-05-18T13:06:04.990+0000] {subprocess.py:93} INFO - task 2
[2024-05-18T13:06:04.992+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-18T13:06:05.047+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=seceond_task, execution_date=20240501T000000, start_date=20240518T130604, end_date=20240518T130605
[2024-05-18T13:06:05.356+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-18T13:06:05.406+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
